jbpm-test-coverage/src/main/java/org/jbpm/test/casemgmt/UpdateTimerCommand.java

```diff
@@ -0,0 +1,86 @@
+package org.jbpm.test.casemgmt;
```

Comment:
Please add licence header to this file (and other as well). Template can be found at https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/ide-configuration/LICENSE-ASL-2.0-HEADER.txt